packages/server/src/Socket.ts

```diff
@@ -31,5 +31,10 @@ export const createSocket = (server: HTTPServer) => {
     socket.on(RoomEvent.Join, async (roomName: string, username: string) => {
       await joinRoom(io, socket, store, roomName, username);
     });
+
+    socket.on(GameEvent.UpdateState, (roomName: string, gameState: GameState) => {
+      store.updateRoomState(roomName, gameState);
+      socket.emit(GameEvent.CurrentState, gameState);
```

Comment:
```suggestion
      io.in(roomName).emit(GameEvent.CurrentState, gameState);
```

Reply:
I see right. I didn't grasp the room thingy! Thanks. We ha a working game now!